sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/util/BigQueryTableInserter.java

```diff
@@ -273,6 +296,144 @@ public void insertAll(TableReference ref, List<TableRow> rowList,
   }
 
   /**
+   * Import files into BigQuery with load jobs.
+   */
+  public void load(
+      String jobId,
+      TableReference ref,
+      List<String> gcsUris,
+      TableSchema schema,
+      WriteDisposition writeDisposition,
+      CreateDisposition createDisposition) throws InterruptedException, IOException {
+    JobConfigurationLoad loadConfig = new JobConfigurationLoad();
+    loadConfig.setSourceUris(gcsUris);
+    loadConfig.setDestinationTable(ref);
+    loadConfig.setSchema(schema);
+    loadConfig.setWriteDisposition(writeDisposition.name());
+    loadConfig.setCreateDisposition(createDisposition.name());
+    loadConfig.setSourceFormat("NEWLINE_DELIMITED_JSON");
+
+    String projectId = ref.getProjectId();
+    for (int i = 0; i < MAX_RETRY_LOAD_JOBS; ++i) {
+      BackOff backoff = new AttemptBoundedExponentialBackOff(
+          MAX_LOAD_JOB_RPC_ATTEMPTS, INITIAL_LOAD_JOB_RPC_BACKOFF_MILLIS);
+      String retryingJobId = jobId + "-" + i;
+      insertLoadJob(retryingJobId, loadConfig, Sleeper.DEFAULT, backoff);
```

Comment:
no
pollJobStatus() blocks until the job is done(SUCCEEDED or FAILED) or it exceeds max retries (UNKNOWN)